### PR TITLE
update gradle and gradle plugin

### PR DESCRIPTION
**Pull request by IlyaEremin**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+org.gradle.daemon="false".equals(System.getenv("TRAVIS"))
```
Comment (adelnizamutdinov) [inline gradle.properties]: I think those should be user specific (parallel is still flaky sometimes and one might not want to have it enabled)
same goes to daemon and jvm heap size

Comment (IlyaEremin) [inline gradle.properties]: Nice to meet you.
Sure, you are right. I fix it tomorrow.

Comment (IlyaEremin) [inline gradle.properties]: Maybe...

**Comment by IlyaEremin** [comment]
deprecated, start another branch with BIG CHANGES

**Pull request closed without merging**